md/rest-api-authorization.md

```diff
@@ -24,9 +24,12 @@ Here are the key points of the authorizations configuration :
 If you are using the standard Portal, you do not need to configure anything.
 * If you want to add extra authorization restrictions based on business rules, turn on the dynamic checks that you want.
 The configuration file defines standard rules for the most frequent cases, so all you need to do is uncomment the rules you want to apply.
+* When you add a custom page or REST API extension, the appropriate resource permission mappings is created. Be careful to not [push configuration](BonitaBPM_platform_setup.md#update_platform_conf) 
```

Comment:
```suggestion
* When you add a custom page or REST API extension, the appropriate resource permission mappings is created. Be careful not to [push a configuration](BonitaBPM_platform_setup.md#update_platform_conf) that has been updated before the addition. This would overwrite the deployed configuration. To avoid doing so, pull the latest configuration, edit the needed elements and push it back to the Bonita runtime in a pretty short amount of time.